cpt-quickstart.rst

```diff
@@ -87,6 +87,9 @@ To install, proceed according to the following steps:
     the Stratum 1 web servers (more on :ref:`proxy configuration here <cpt_squid>`). For Cern
     repositories, the Stratum 1 web servers are listed in
     /etc/cvmfs/domain.d/cern.ch.conf.
+    
+    If you're unsure about the proxy names, set
+    ``CVMFS_HTTP_PROXY=DIRECT``.
```

Comment:
I agree with the caveat - I was thinking about it from the perspective of a single user but of course that would need to be made clear. It would also need to be specified in the Mac OS X instructions, although maybe there we could assume that only single users would be setting it up on OS X as it is a less common operating system for servers or infrastructure.